src/expandable/multi_request.rs

```diff
@@ -53,6 +56,32 @@ mod tests {
     assert_eq!(benchmark.len(), 3);
   }
 
+  #[test]
+  fn expand_with_times_option() {
+    let text = "---\nname: foobar\nrequest:\n  url: /api/{{ item }}\ntimes: 2\nwith_items:\n  - 1\n  - 2\n  - 3";
+    let docs = yaml_rust::YamlLoader::load_from_str(text).unwrap();
+    let doc = &docs[0];
+    let mut benchmark: Benchmark = Benchmark::new();
+
+    expand(doc, &mut benchmark);
+
+    assert_eq!(is_that_you(doc), true);
+    assert_eq!(benchmark.len(), 2);
+  }
+
+  #[test]
+  fn expand_with_times_option_greater_than_with_items_list() {
```

Comment:
Maybe it's interesting to assert the performed IDs for this test case

Reply:
Could you give me a hint on how to assert them?  
I just have the `Vec<Runnable>` available, which only has an `execute` function. Can I get access to the used item somehow?

Reply:
Mmmmm this is true. It is behind the interface. Leave it like this and we can thing about this later